modules/intro/pages/index.adoc

```diff
@@ -80,9 +80,9 @@ for the most up-to-date news on pyTigerGraph.
 |===
 
 == Peformance Notes
-pyTigerGraph is a feature rich package that allows you to interact with TigerGraph databases from Python.
-However, it is important to note that there will be a performance overhead when using pyTigerGraph compared to using cURL-like requests directly to the TigerGraph REST++ API.
-This is because pyTigerGraph is a Python package that wraps the REST++ API and adds additional functionality such as URL building, logging, and authentication handling, as well as error checking.
+pyTigerGraph is a feature-rich package that allows you to interact with TigerGraph databases using Python.
+It enhances the REST++ API by adding extra features, such as URL setup, log creation, authentication, and error checks.
```

Comment:
```suggestion
It enhances TigerGraph's REST API  by adding extra features, such as URL setup, log creation, authentication, and error checks.
```
REST is a well-known industry concept. REST++ is the name of a TigerGraph-specific component, which handles only a subset of TigerGraph's REST API requests.